roles/network_plugin/calico/defaults/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+# the default value of name
+calico_cni_name: k8s-pod-network
```

Comment:
Please add this in the sample config in `inventory/sample/` as well so it is visible to any deployer.

Reply:
Thanks! Done.